helm/template-operator/templates/pull-secret.yaml

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Why is there a pull secret in some repos and not others? Only needed for private images no?

Reply:
I am not sure if the operators in operatorweek will be public or private, so I thought this would be safer.